pkgs/by-name/an/anubis/package.nix

```diff
@@ -58,12 +76,12 @@ buildGoModule (finalAttrs: {
   ]
   ++ lib.optionals stdenv.hostPlatform.isLinux [ "-extldflags=-static" ];
 
-  postPatch = ''
-    patchShebangs ./web/build.sh
-  '';
-
   preBuild = ''
```

Comment:
I think we technically run this in the go vendoring phase which is not great for long term reproducibility and it only works because we patch out the one go generate command.

I do not have a better idea than checking $name for containing go-module. See my Ani is PR for a complete code example.